bin/check-http-json.rb

```diff
@@ -74,6 +74,11 @@ class CheckJson < Sensu::Plugin::Check::CLI
   option :dump_json, short: '-d', long: '--dump-json', boolean: true, default: false
   option :pretty, long: '--pretty', boolean: true, default: false
 
+  option :response_code,
+         long: '--response-code REGEX',
+         description: 'Check for a specific response code REGEX',
```

Comment:
This does not convey if the matching or unmatched HTTP status codes will have an OK status. Can we add something to clarify?